### ISSUE TITLE
A bug regarding text numbers in data_utils.py

### ISSUE THREAD

**Issue by frankxu2004** [harvardnlp/data2text#13]
In line 134 and 136, it should be `and annoying_number_word(sent, i)` rather than `and not annoying_number_word(sent, i)`, since the function returns True if it's not saying three-point related stuff.

**Comment by swiseman** [comment]
Yes, you're right (and this was also pointed out here: https://github.com/harvardnlp/data2text/issues/6 ). I'm going to leave it as is for reproducibility, but feel free to change in your fork.
